aries-site/src/examples/components/skeleton/SimpleSkeleton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Box, Text, Skeleton } from 'grommet';
+
+export const SkeletonExample = () => (
```

Comment:
You kind of hint at it in the Anatomy table, but when you do examples, keep in mind that we're trying to avoid folks using the `<Skeleton/>` component directly. Instead we're hoping they're dashboard cards or other components can just be skeleton-ized by simply using the 'skeleton' prop to 'Box' or 'Card' and let the contained 'Text', 'Heading', etc to render themselves as the gray box.